### ISSUE TITLE
There is a big difference between the training loss and valid loss

### ISSUE THREAD

**Issue by Sandydreamsmile** [facebookresearch/svoice#7]
After I trained the model with the provided samples, I found that the valid loss is far good than the training loss. 
such as below:

    "train": -6.68186240196228,
    "valid": -15.98158609867096,
    "best": -15.98158609867096

Then I use my own audio dataset, I also found this situation after epoch 8th.
Is this situation is reasonable?

**Comment by adiyoss** [comment]
Hi @Sandydreamsmile, 
Yes, this is expected since during training we optimize multiple loss functions, so the value that you see here is the sum over all loss functions. 
During the evaluation, we report the loss using the last layer only. 

As a general note (just to make sure we are on the same page), the provided samples are just a small toyish set to get the model and the whole framework running. If you want to train a good performing model, I suggest using bigger datasets such as WHAMR!, LibriMix (augmented with some reverb), etc.

**Comment by Sandydreamsmile** [comment]
Thank you for explanation! I will try with a  bigger dataset!
